### PR TITLE
SONAR-8837 log indexing stats

### PR DESCRIPTION
**Pull request by bartfastiel**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/es/IndexerStartupTask.java [hunk 1]
```diff
@@ -56,20 +57,20 @@ private boolean indexesAreEnabled() {
   private void indexEmptyTypes(StartupIndexer indexer) {
     Set<IndexType> emptyTypes = getEmptyTypes(indexer);
     if (!emptyTypes.isEmpty()) {
-      log(indexer, emptyTypes, "...");
+      Profiler profiler = Profiler.create(LOG);
+	  profiler.startInfo(getLogMessage(emptyTypes, "..."));
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/es/IndexerStartupTask.java]: MAJOR Make this line start at column 7. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [IndexerStartupTask.java](https://github.com/SonarSource/sonarqube/blob/71d7d3bfe2d42cb3a2b932546c074410d79fafc4/server/sonar-server/src/main/java/org/sonar/server/es/IndexerStartupTask.java): Replace all tab characters in this file by sequences of white-spaces. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00105)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by bartfastiel** [comment]
Makes the logs look like this:

`2017.02.28 08:11:29 INFO  web[][o.s.s.e.IndexerStartupTask] Full indexing of type [issues/issue] ...`

`2017.02.28 08:11:29 INFO  web[][o.s.s.e.IndexerStartupTask] Full indexing of type [issues/issue] done | time=33ms`

`2017.02.28 08:11:29 INFO  web[][o.s.s.e.IndexerStartupTask] Full indexing of types [components/authorization],[projectmeasures/authorization],[issues/authorization] ...`

`2017.02.28 08:11:29 INFO  web[][o.s.s.e.IndexerStartupTask] Full indexing of types [components/authorization],[projectmeasures/authorization],[issues/authorization] done | time=12ms`

File: server/sonar-server/src/main/java/org/sonar/server/es/IndexerStartupTask.java [hunk 2]
```diff
@@ -56,20 +57,20 @@ private boolean indexesAreEnabled() {
...
     return indexer.getIndexTypes().stream().filter(esClient::isEmpty).collect(toSet());
   }
 
-  private void log(StartupIndexer indexer, Set<IndexType> emptyTypes, String suffix) {
+  private String getLogMessage(Set<IndexType> emptyTypes, String suffix) {
     String s = emptyTypes.size() == 1 ? "" : "s";
     String typeList = emptyTypes.stream().map(Object::toString).collect(Collectors.joining(","));
-    String indexerName = indexer.getClass().getSimpleName();
-    LOG.info("Full indexing of type{} {} using {} {}", s, typeList, indexerName, suffix);
+    return String.format("Full indexing of type%s %s %s", s, typeList, suffix);
```
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/es/IndexerStartupTask.java]: I think without "Full" would be better.

**Pull request merged**